src/components/shared/ExternalLink.tsx

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 ExternalLink.propTypes = {
     children: PropTypes.any.isRequired,
     link: PropTypes.string.isRequired,
+    disableHover: PropTypes.bool,
```

Comment:
Leaning towards renaming this prop to `hideOverlay`. To me, it more clearly communicates what is changed when the prop is set.

Reply:
I agree... I think `hideOverlay` is a better name.